src/common/io.ts

```diff
@@ -824,6 +824,39 @@ export class IOWriteable extends IOReadable implements AoIOWrite {
     params: {
       target: string;
       decreaseQty: number | mIOToken;
+      instant: boolean;
+    },
+    options?: WriteOptions,
+  ): Promise<AoMessageResult> {
+    const { tags = [] } = options || {};
+    if (params.instant == true) {
+      return this.process.send({
+        signer: this.signer,
+        tags: [
+          ...tags,
+          { name: 'Action', value: 'Decrease-Delegate-Stake' },
+          { name: 'Target', value: params.target },
+          { name: 'Quantity', value: params.decreaseQty.valueOf().toString() },
+          { name: 'Instant', value: 'true' },
```

Comment:
```suggestion
          { name: 'Instant', value: instant || false },
```

Reply:
look at the other tag combining to avoid this `if` block entirely

Reply:
How bout this? 

https://github.com/ar-io/ar-io-sdk/commit/5e82a6a02f2c19ef647b164b97140d0d7e04240e